docs/using.rst

```diff
@@ -60,6 +60,8 @@ Common |sr| Usage Examples
 
 These examples use curl commands to interact with the |sr| :ref:`API <schemaregistry_api>`.
 
+If you want to see a playbook of interacting with a |sr| configured with security, please refer to the :ref:`Confluent Platform demo<cp-demo>`
```

Comment:
i think this reference should be moved up to line 62 (above introduction to curl examples)
and revise as follows (add space in link, period to sentence, make a tip):

```
.. tip:: For a detailed example that uses |sr| configured with security, see the :ref:`Confluent Platform demo <cp-demo>`.
```